STAC-0: Describe extra step to generate OIDC values to use rancher as… (#55)
rancher/stackstate-product-docs: docs/next/modules/en/pages/k8s-suse-rancher-prime.adoc

The SUSE Observability Values chart generates affinity configurations you can use with the main SUSE Observability chart to control pod scheduling behavior. Refer to xref:/setup/install-stackstate/kubernetes_openshift/affinity.adoc[Configure Kubernetes Affinities]) for more information.
====

> . Configure SUSE Observability for using Rancher as an OIDC provider

Reviewer: ```suggestion
. Configure SUSE Observability to use Rancher as an OIDC provider.
```

> . Configure SUSE Observability for using Rancher as an OIDC provider

Author: I can add a `NOTE` somewhere stressing the importance of this step is you are planning to use RBAC

>     Generate your `oidc_values.yaml`. This guide assumes that you save it in the `$VALUES_DIR`

Reviewer: ```suggestion
    Generate the `oidc_values.yaml`. This guide assumes that you save it in the `$VALUES_DIR`
```
